schemas/purl-test.schema.json

```diff
@@ -284,6 +295,31 @@
               "expected_failure_reason"
             ]
           }
+        },
+        {
+          "if": {
+            "properties": {
+              "test_type": { "const": "validation" }
+            },
+            "required": ["test_type"]
+          },
+          "then": {
+            "properties": {
+              "input": {
+                "title": "Expected decoded PURL components",
+                "description": "Test input as an object with decoded PURL components for validation.",
+                "$ref": "#/definitions/purl_components"
+              },
+              "expected_messages": {
+                "title": "Expected messages",
+                "description": "Validation messages expected for this test. Can be null or a list of strings.",
```

Comment:
Are implementations really expected to produce exactly these text messages?

Reply:
@matt-phylum re:
> Are implementations really expected to produce exactly these text messages?


good point. I have mixed feeling on that alright. @TG1999 let's move these kind of tests to a separate PR, for separate discussion.